docs/recipes/watch-mode.md

```diff
@@ -59,11 +59,9 @@ Please note that the TAP reporter is unavailable when using watch mode.
 
 ## Requirements
 
-AVA uses [`chokidar`] as the file watcher. It's configured as an optional dependency since `chokidar` sometimes can't be installed. Watch mode is not available if `chokidar` fails to install, instead you'll see a message like:
+AVA uses [`chokidar`] as the file watcher.
 
-> The optional dependency chokidar failed to install and is required for --watch. Chokidar is likely not supported on your platform.
-
-Please refer to the [`chokidar` documentation][`chokidar`] for how to resolve this problem.
+Please refer to the *[Install Troubleshooting]* section of `chokidar` documentation for how to resolve the installation problems with chokidar.
```

Comment:
I would add a note that even if you see warnings about optional dependencies failing, it will still work fine. People have tendency to just blindlessly open issues when they see a warning of any kind.